dsf/org.eclipse.cdt.dsf.ui/src/org/eclipse/cdt/dsf/debug/internal/ui/disassembly/model/SourceFileInfo.java

```diff
@@ -147,6 +147,10 @@ public String getLines(int first, int last) {
 			}
 			return fSource.get(startOffset, endOffset - startOffset);
 		} catch (BadLocationException e) {
+			// Log error to indicate what is tha cause of the issue
+			String warningMessage = "Line(s) " + Integer.toString(first) + "-" + Integer.toString(last) //$NON-NLS-1$
+					+ " cannot be found in file: " + fFileKey + ".\nSkipping lines!"; //$NON-NLS-1$
+			DsfUIPlugin.logErrorMessage(warningMessage); //$NON-NLS-1$
```

Comment:
This logErrorMessage has two minor problems:

1. It discards the exception. I think you should log the exception too, it provides the stack trace/context to diagnose the problem
2. What is logged in this case is `Internal message logged from Debug UI` as type `INTERAL_ERROR`, which seems a little to harsh here.

```suggestion
			DsfUIPlugin.log(Status.warning(warningMessage, e));
```

Reply:
I updated this and the Bundle-Version. Thank you!